### PR TITLE
Customizable search filters on the search results page, category filter works

### PR DESCRIPTION
**Pull request by SmileyFace799**
I have a 7-hour car ride ahead of me so I was a bit in a hurry with this one, please review carefully as this may contain oopsies

### CODE REVIEW

File: backend/src/main/java/no/ntnu/database/controller/CategoryController.java [hunk 1]
```diff
@@ -56,21 +56,20 @@ public Iterable<Category> getAllCategories() {
...
 	//TODO: Improve swagger here with api responses.
-	@ApiResponse(responseCode = "200", description = "Successfully searched")
-	@GetMapping(value = "{query}", produces = {"application/json"})
-	public Iterable<Category> searchCategory(@PathVariable String query) {
-		return query == null || query.isBlank()
-				? service.getAllCategories()
-				: service.searchCategory(query);
+	@ApiResponse(responseCode = "200", description = "Found category")
+	@GetMapping(value = "{id}", produces = {"application/json"})
+	public ResponseEntity<Category> searchCategory(@PathVariable int id) {
```
Comment (Kenzirey) [inline backend/src/main/java/no/ntnu/database/controller/CategoryController.java]: If you're no longer searching, should this perhaps be renamed to getCategory instead?

**Review by Kenzirey — CHANGES_REQUESTED** [review]

File: frontend/src/plugins/vuetify.js [hunk 2]
```diff
@@ -28,7 +28,8 @@ const lightTheme = {
     info: '#03a9f4',
     success: '#8bc34A',
     text: '#000000',
-    weakText: "aaaaaa"
+    weakText: "aaaaaa",
+    separator: "eeeeee"
```
Comment (Kenzirey) [inline frontend/src/plugins/vuetify.js:32]: this gave me a giggle

Comment (SmileyFace799) [reply frontend/src/plugins/vuetify.js:32]: aaaaaa eeeeee 😎

File: backend/src/main/java/no/ntnu/database/controller/CategoryController.java (hunk 1 above)
Comment (SmileyFace799) [reply backend/src/main/java/no/ntnu/database/controller/CategoryController.java]: Yes, I fixed it :)

**Review by Kenzirey — APPROVED** [review]
Nice!

**Pull request merged**